src/App.tsx

```diff
@@ -1,22 +1,25 @@
+import React from 'react';
 import "./App.css";
-import { IVideoCard } from "./common/interfaces/IVideoCard";
-import { VideoCard } from "./components/molecules/videoCard/VideoCard";
+import { BrowserRouter, Routes, Route } from 'react-router-dom'
+import Home from "./components/templates/Home";
+import Contact from "./components/templates/Contact";
+import VideoListPage from "./components/templates/VideoListPage";
+import SeriesPage from "./components/templates/SeriesPage";
+import AnimationPage from "./components/templates/AnimationPage";
+
 
 function App() {
-  const video: IVideoCard = {
-    url: "https://player.vimeo.com/video/121707380",
-    // url: "https://player.vimeo.com/video/121707380?h=d6c613b55d&color=ef9700&portrait=0",
-    title: "From Mass to the Mountain",
-    clientName: "Self",
-    genre: "documentary",
-    roles: ["director_producer"],
-  };
+
   return (
-    <div className="App">
-      <header className="App-header">
-        <VideoCard {...video} />
-      </header>
-    </div>
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/contact" element={<Contact />} />
+        <Route path="/videoListPage" element={<VideoListPage />} />
```

Comment:
URL conventions suggest using kebab case.

Since urls are technically not case sensitive, this is basically `videolistpage`.


```suggestion
        <Route path="/video-list-page" element={<VideoListPage />} />
```

Reply:
Noted. Will change tomorrow.